packages/react-heartwood-components/src/components/TruncatedList/TruncatedList.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+import React, { Component, Fragment } from 'react'
+import cx from 'classnames'
+
+import Button from '../Button/Button'
+import EmptyState from '../EmptyState/EmptyState'
+import RecordSelectionList from '../RecordSelectionList/RecordSelectionList'
+
+import type { RecordSelectionListItemProps } from '../RecordSelectionList/RecordSelectionList'
+
+type Props = {
```

Comment:
Please add prop descriptions. I know this isn't explicitly in the A/C's, but I think we should include these descriptions any time we add a new component to facilitate collaboration.

Reply:
Good call. was an oversight as I was architecting out the props and meant to come back to it.